src/components/SearchPerson/SearchPerson.tsx

```diff
@@ -58,7 +58,7 @@ const SearchPerson: React.FunctionComponent<Props> = ({ searchQuery }) => {
     PERSON_GQL,
     {
       errorPolicy: 'all',
-      variables: { query: searchQuery, cursor: cursor }
+      variables: { query: searchQuery.replace(/^https?:\/\//, ''), cursor: cursor }
```

Comment:
Here issue was with the `://` characters inside the URL. Hence we just replaced the `http://` or `https://` from the string.

Yes, it will work for all following scenarios. It will give similar result for all.

1. 0000-0001-6528-2027 
2. orcid.org/0000-0001-6528-2027 
3. https://orcid.org/0000-0001-6528-2027
4. http://orcid.org/0000-0001-6528-2027
5. https orcid.org/0000-0001-6528-2027 (without `://` will also work)